frontend/src/components/organisms/Header.tsx

```diff
@@ -1,24 +1,25 @@
-import React, { useState, useEffect, FC } from "react";
+import React, { useState, FC } from "react";
 import headerLogo from "../atoms/logo.png";
 import ToggleButton from "../molecules/ToggleButton";
 import Navigation from "../molecules/Navigation";
 
 const Header: FC = () => {
-  /*TODO: Implement a proper state management once firebase auth is implemented*/
-  const [loggedIn, setLoggedIn] = useState<boolean>(true);
   const [open, SetOpen] = useState(false);
   const toggleFunction = () => {
     SetOpen((prevState) => {
       return !prevState;
     });
   };
 
+  const session: null|string = sessionStorage.getItem("isLoggedIn");
+  const isLoggedIn = true ? session != null : false;
+
   return (
     <div className="header drop-shadow-md flex">
       <a href="/" className="px-5">
         <img src={headerLogo} alt="melBee_logo" className="py-2" width="80" />
       </a>
-      {loggedIn ? (
+      {isLoggedIn ? (
```

Comment:
if there's nothing to show otherwise, I guess its better to use `{isLoggedin && ...}` than the if else statement with ? and :?

Reply:
I'm sorry for the late response

I was suggesting we use the `&&` conditional rendering!
If we use `&&`, we don't need to have `null`
[](https://reactjs.org/docs/conditional-rendering.html)

Reply:
I have edited the file, please double check for me, thank you!